BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift

```diff
@@ -122,82 +122,11 @@ public final class GiniBankConfiguration: NSObject {
     
     // MARK: Camera options
     
-    /**
-     Sets the color of camera preview corner guides.
-     */
-    @objc public var cameraPreviewCornerGuidesColor = UIColor.white
-    
-    /**
-     Sets the background color of camera container view.
-     */
-    @objc public var cameraContainerViewBackgroundColor = GiniColor(light: .black, dark: .black)
-    
-    /**
-     Sets the color of camera preview frame.
-     */
-    @objc public var cameraPreviewFrameColor = GiniColor(light: UIColor(white: 0.0, alpha: 0.7), dark: UIColor(white: 0.0, alpha: 0.7))
-    
-    /**
-     Sets the background color of camera buttons view.
-     */
-    @objc public var cameraButtonsViewBackgroundColor = GiniColor(light: .black, dark: .black)
-    
     /**
      Set the types supported by the file import feature. `GiniCaptureImportFileTypes.none` by default.
      */
     @objc public var fileImportSupportedTypes = GiniConfiguration.GiniCaptureImportFileTypes.none
-    
-    /**
-     Sets the background color of the new file import button hint.
-     */
-    @objc public var fileImportToolTipBackgroundColor = UIColor.white
-    
-    /**
-     Sets the text color of the new file import button hint.
-     */
-    @objc public var fileImportToolTipTextColor = UIColor.black
-    
-    /**
-     Sets the close button color of the new file import button hint.
-     */
-    @objc public var fileImportToolTipCloseButtonColor = Colors.Gini.grey
-    
-    /**
-     Sets the background style when the tooltip is shown.
-     */
-    public var toolTipOpaqueBackgroundStyle: OpaqueViewStyle {
-        
-        set {
-            _toolTipOpaqueBackgroundStyle = newValue
-        }
-        
-        get {
-            
-            if let setValue = _toolTipOpaqueBackgroundStyle {
-                return setValue
-            } else {
-                
-                if #available(iOS 13.0, *) {
-                    return .blurred(style: .regular)
-                } else {
-                    return .blurred(style: .dark)
-                }
-            }
-        }
-    }
-    
-    private var _toolTipOpaqueBackgroundStyle: OpaqueViewStyle?
-    
-    /**
-     Sets the text color of the item selected background check.
-     */
-    @objc public var galleryPickerItemSelectedBackgroundCheckColor = Colors.Gini.blue
-    
-    /**
-     Sets the background color for gallery screen.
-     */
-    @objc public var galleryScreenBackgroundColor = GiniColor(light: .black, dark: .black)
```

Comment:
@vizaknaidavid could you please check if instead of `galleryScreenBackgroundColor` we use the GiniColors pallet?

Reply:
Yes, it is used. 
`view.backgroundColor = GiniColor(light: .GiniCapture.light2, dark: .GiniCapture.dark2).uiColor()`